lib/ex_wire/adapter/tcp.ex

```diff
@@ -13,19 +19,22 @@ defmodule ExWire.Adapter.TCP do
   @doc """
   Starts an outbound peer to peer connection.
   """
-  def start_link(:outbound, host, port, remote_id) do
-    GenServer.start_link(__MODULE__, %{is_outbound: true, host: host, port: port, remote_id: remote_id, active: false})
+  def start_link(:outbound, host, port, remote_id, subscribers \\ []) do
+    remote_id_hex = remote_id |> Binary.drop(1) |> ExthCrypto.Math.bin_to_hex
+    ident = Binary.take(remote_id_hex, 6) <> "..." <> Binary.take(remote_id_hex, -6)
```

Comment:
Would it make sense to add a  `Peer` struct and move this function in as the `to_string` function?